alvr/server_core/src/statistics.rs

```diff
@@ -271,13 +271,14 @@ impl StatisticsManager {
                 self.packets_lost_partial_sum = 0;
             }
 
-            // While not accurate, this prevents NaNs and zeros that would cause a crash or pollute
-            // the graph
-            let bitrate_bps = if network_latency != Duration::ZERO {
-                frame.video_packet_bytes as f32 * 8.0 / network_latency.as_secs_f32()
-            } else {
-                0.0
-            };
+            let packet_bits = frame.video_packet_bytes as f32 * 8.0;
+            let throughput_bps = packet_bits
+                / Duration::max(network_latency, Duration::from_millis(1)).as_secs_f32();
```

Comment:
1 millisecond seems too high, you might be able to go below that with wired (tho probably not and I'm not sure but I think you should make it something like 0.5 ms instead (or maybe then it'll throw it even more off, dunno))

Reply:
This is just a sanitization step to avoid dividing by zero. We cause use `from_micros(1)` instead.

Reply:
Well either we keep it at a sane value so that if something goes wrong the values stay reasonable or we just null them, if we were to actually hit that we'd create insane values that'd just cause issues in the graphs. Think we should do the latter tho